scripts/ci_build_cairo.py

```diff
@@ -0,0 +1,209 @@
+# Logic is as follows:
```

Comment:
Should this file perhaps live in, say, `.github/scripts`? Practically, it's only for CI and not something one should use locally, I guess. 🤔

Reply:
Yeah, thinking of it, I think I'll move this to the other folder.